# Actual vacuum level
bool vacuum

# Actual threshold
float32 threshold